packages/attention/src/component.tsx

```diff
@@ -245,33 +317,4 @@ export function Attention(props: AttentionProps) {
   )
 }
 
-const arrowDirectionClassname = (dir: string) => {
-  const direction = dir.charAt(0).toUpperCase() + dir.slice(1)
-
-  return `arrowDirection${direction}`
-}
-
-const Arrow = forwardRef<HTMLDivElement, ArrowProps>(({ direction, ...rest }, ref) => {
-  const arrowDirection = opposites[direction]
-
-  const arrowClasses = classNames(
-    ccAttention.arrowBase,
-    ccAttention[arrowDirectionClassname(arrowDirection)],
-    variantClasses[getVariant(rest)].arrow
-  )
 
-  return (
-    <div
-      ref={ref}
-      className={arrowClasses}
-      style={{
-        // TW doesn't let us specify exactly one corner, only whole sides
-        borderTopLeftRadius: '4px',
-        zIndex: 1,
-        // border alignment is off by a fraction of a pixel, this fixes it
-        [`margin${arrowDirectionClassname(arrowDirection)}`]: '-0.5px',
```

Comment:
Removing negative margin may result in borders not aligning:
<img width="342" alt="Screenshot 2024-03-08 at 13 36 20" src="https://github.com/warp-ds/react/assets/41303231/04608f50-968f-40ef-a386-b61fa563f4ea">

Reply:
hmm true! I'll test to revert the removed code! :)

Reply:
I have now reverted this change, but I decided to move `arrowDirectionClassname` into the core-library instead, since that one is exactly the same in react, vue and elements. And I have also moved in this `[`margin${arrowDirectionClassname(arrowDirection)}`]: '-0.5px',` to the core-library, so we just need to add it once! :)